extensions/agent/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "agent",
   "displayName": "SQL Server Agent",
   "description": "Manage and troubleshoot SQL Server Agent jobs",
-  "version": "0.46.0",
+  "version": "0.47.0",
```

Comment:
Don't update this - it's already been bumped. The new flow is that extensions are bumped immediately after they're released (similar to how we do it for ADS)